### DISCUSSION TITLE
Connect data to PowerBi

### DISCUSSION THREAD

**Discussion by patpiet** [dcaribou/transfermarkt-datasets#122] [Q&A]
Hi!

Great work with creating this dataset!

I would like to kindly ask if you know what would be the best way to connect this database to PowerBI. 
My goal is to update the dashboard automatically as you update the data.

**Comment by dcaribou** [comment]
Hi @patpiet,

There's a Power BI integration for data.world. Maybe that'll do it for you?

https://data.world/integrations/powerbi

**Reply by patpiet** [reply]
Yeah I have actually tried it. The thing is that the data exceeded 100mb. After that you can't connect to two tables from dataset:/

**Comment by dcaribou** [comment]
What if the data was in Google Drive, would that help?

I noticed there's a connector for GDrive
https://learn.microsoft.com/en-us/connectors/googledrive/

**Reply by patpiet** [reply]
Yes, if it had public access I can see it can work.

It would be amazing if you could do this 👍
